internal/chart/testdata/charts/dashboard-traefik-secret.yaml

```diff
@@ -0,0 +1,387 @@
+---
```

Comment:
This test file is the same as `dashboard-traefik` with addition of a `ClusterIssuer` at line 339, which is generated because the tested `App` has a populated `SecretName`.